app/models/maintenance_tasks/csv_collection_builder.rb

```diff
@@ -16,13 +16,13 @@ def collection(task)
     end
 
     # The number of rows to be processed. Excludes the header row from the
```

Comment:
We can stop mentioning the part about the header, since it's part of using the CSV library for an accurate row count.

Reply:
Removed